src/Symfony/Component/Notifier/Bridge/Teams/TeamsOptions.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Teams;
+
+use Symfony\Component\Notifier\Bridge\Teams\Action\TeamsActionInterface;
+use Symfony\Component\Notifier\Bridge\Teams\Section\TeamsSection;
+use Symfony\Component\Notifier\Bridge\Teams\Section\TeamsSectionInterface;
+use Symfony\Component\Notifier\Exception\InvalidArgumentException;
+use Symfony\Component\Notifier\Exception\LogicException;
+use Symfony\Component\Notifier\Message\MessageOptionsInterface;
+use Symfony\Component\Notifier\Notification\Notification;
+
+/**
+ * @author Edouard Lescot <edouard.lescot@gmail.com>
+ *
+ * @experimental in 5.3
```

Comment:
```suggestion
```

Reply:
Are you sure about this ? @fabpot  asked to add @experimental in 5.3 in all classes